js/foundation.drilldown.js

```diff
@@ -298,13 +298,14 @@ class Drilldown {
    * @private
    */
   _getMaxDims() {
-    var max = 0, result = {};
-    this.$submenus.add(this.$element).each(function(){
-      var numOfElems = $(this).children('li').length;
-      max = numOfElems > max ? numOfElems : max;
+    var $uls = this.$element.find('ul'), max = 0, result = {};
+    $uls.add(this.$element).each(function(){
+      var $elem = $(this);
+      $elem.hasClass('is-drilldown-submenu')?$elem.css({height:'auto'}):'';
+      max = Math.max(this.getBoundingClientRect().height, max);
+      $elem.hasClass('is-drilldown-submenu')?$elem.css({height:'100%'}):'';
```

Comment:
This doesn't seem right - what if someone has overridden the styles on the submenus to have a different height?

This loop approach seems right - the bug seems to be due to assuming all of the menu items will always be the same size, where actually should take a look - but this auto/100% hardcoding feels wrong.  When these are actually shown, its done via adding a class, no?  Can we use that as a way to get them to expand to their auto height instead, and find from that what the height should be?